opensrp-app/src/main/java/org/smartregister/sync/intent/SyncIntentService.java

```diff
@@ -208,6 +200,30 @@ private synchronized void fetchRetry(final int count, boolean returnCount) {
             fetchFailed(count);
         }
     }
+    protected Response getUrlResponse(SyncConfiguration configs,HTTPAgent httpAgent, String initialUrl,Long lastSyncDatetime ,boolean returnCount){
```

Comment:
We can add unit tests for this to cover the **IF**, the **ELSE** and the **Exception** thrown here